package.json

```diff
@@ -35,7 +35,7 @@
     "@emotion/react": "11.11.1",
     "@emotion/styled": "11.11.0",
     "@graasp/chatbox": "2.0.1",
-    "@graasp/query-client": "1.4.3",
+    "@graasp/query-client": "github:graasp/graasp-query-client#423-add-etherpad-queue",
```

Comment:
TODO: update to actual version when https://github.com/graasp/graasp-query-client/pull/424 is merged